tests/testthat/test-growth_rate.R

```diff
@@ -0,0 +1,14 @@
+library(dplyr)
+
+x <- jhu_csse_daily_subset %>%
+  select(-case_rate_7d_av,-death_rate_7d_av, -cases_7d_av) %>%
+  group_by(geo_value) %>% 
+  mutate(cases_gr = growth_rate(x = time_value,  y = cases))
+
+test_that("Test error throwing",{
+  expect_error(growth_rate(x=1:3,y=1:4),
+               "`x` and `y` must have the same length.")
+  expect_error(growth_rate(x=1:20,y=1:20,x0=21),
+               "`x0` must be a subset of `x`.")
```

Comment:
Please place a comment here (between the error-expected and no-error-expected tests) explaining that the statements below actually test that there is no error.  (Or, if you think it's clearer, you don't need `expect_error(<stuff>, NA)` at all; you could just write `<stuff>` and I believe testthat has roughly similar behavior.)

Reply:
I've just had a habit of doing `expect_error(<stuff>, NA)` as having `<stuff>` makes it unclear if it's just random code I forgot to delete or actual testing.

Reply:
+1 plus this also (I think) tallies it as its own test, which makes sense.  I still think an explanatory comment or putting the NA on a separate line is needed to more prominently highlight that this is non-error behavior.

Reply:
I don't see any tests here that are expected to NOT error, so I'm assuming this comment has been resolved/non-error tests have been moved elsewhere.